sympy/sets/sets.py

```diff
@@ -1667,6 +1709,10 @@ def is_EmptySet(self):
     def _measure(self):
         return 0
 
+    @property
+    def _kind(self):
+        return SetKind(UndefinedKind)
```

Comment:
Need to think about what to do for EmptySet. Maybe it could just be SetKind() or something.

Reply:
should `UniversalSet` should also be SetKind()